src/cbmc/Makefile

```diff
@@ -53,6 +53,14 @@ CLEANFILES = cbmc$(EXEEXT)
 
 all: cbmc$(EXEEXT)
 
+ifeq ($(BUILD_ENV_),MSVC)
+  CP_CXXFLAGS += /DCBMC_TAG_VERSION='"""$(CBMC_VERSION)"""' \
+       /DCBMC_VERSION='"""($(GIT_INFO))"""'
```

Comment:
Why is this different from https://github.com/diffblue/cbmc/pull/668/files#diff-d9c95ec27fc25a0481354de531adad56R34?